src/main/java/emu/grasscutter/database/DatabaseHelper.java

```diff
@@ -154,8 +155,8 @@ public static synchronized void deleteAccount(Account target) {
         DatabaseManager.getAccountDatastore().find(Account.class).filter(Filters.eq("id", target.getId())).delete();
     }
 
-    public static List<Player> getAllPlayers() {
-        return DatabaseManager.getGameDatastore().find(Player.class).stream().toList();
+    public static Stream<Player> getAllPlayers() {
```

Comment:
Sadly, this is an API break. This will need to be a new method, and unfortunately it can't share the name `getAllPlayers`. Since the name conflict is a problem anyway, it might be better to just make something like:

```java
public static <T> Stream<T> getByGameClass(Class<T> classType) {
    return DatabaseManager.getGameDatastore().find(classType).stream();
}
```

Reply:
ok

Reply:
By the way, I recommend annotating the method as deprecated and explaining why.

Reply:
Yeah, `@Deprecated(forRemoval = true)` will do it.